{{cookiecutter.project_slug}}/app.json

```diff
@@ -25,26 +25,33 @@
     "web": {
       "quantity": 1,
       "size": "free"
+    },
+    "dev_worker": {
+      "quantity": 1,
+      "size": "free"
+    },
+    "worker": {
+      "quantity": 0,
+      "size": "free"
+    },
+    "scheduler": {
+      "quantity": 0,
+      "size": "free"
     }
   },
   "image": "heroku/python",
   "addons": [
-    "heroku-postgresql"
+    "heroku-postgresql",
+    "heroku-redis",{% if cookiecutter.use_bucketeer_aws_for_file_storage == 'y' %}
+    "bucketeer",{% endif %}
+    "newrelic",
+    "sentry"
   ],
   "buildpacks": [
-      {"url": "heroku/node"},
+      {"url": "heroku/nodejs"},
       {"url": "heroku/python"}
   ],
-  "environments": {
-    "test": {
-      "scripts": {
-        "test-setup": "pip install --upgrade -r requirements/test.txt",
-        "test": "pytest"
-      },
-      "env": {
-        "DJANGO_SETTINGS_MODULE": "config.settings.test",
-        "DATABASE_URL": "sqlite:///test.db"
-      }
-    }
```

Comment:
I removed this because it's broken... We could certainly get it working (by porting over some of the CircleCI config and env vars) if we want to support Heroku CI, but "here and broken" seemed worse that "not supported".